src/js/components/Tip/Tip.js

```diff
@@ -0,0 +1,81 @@
+import React, {
+  cloneElement,
+  forwardRef,
+  useContext,
+  useState,
+  useRef,
+} from 'react';
+import { ThemeContext } from 'styled-components';
+
+import { defaultProps } from '../../default-props';
+import { Box } from '../Box';
+import { Drop } from '../Drop';
+
+const Tip = forwardRef(({ children, content, dropProps }, tipRef) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+  const [over, setOver] = useState(false);
+
+  // console.log(tipRef);
+  const ref = useRef();
+  const componentRef = tipRef || ref;
+
+  const wrapChildIfString = child => {
+    // Handle the use case of a primitive string child
+    // so we'll be able to assign ref and events on the child.
+    // console.log(child);
+    return typeof child === 'string' ? <span>{child}</span> : child;
+  };
+
+  const clonedChild = React.Children.map(children, child =>
```

Comment:
Can we enforce that we only have a single child, like Keyboard.js does? See: `Children.only(children)`

Reply:
Good comment. I was just having the exact same thought on my walk. The problem with more than one child is that the tip will only be shown on the latter.